doc-content/enterprise-only/admin-and-config/deployment-descriptors-con.adoc

```diff
@@ -7,7 +7,7 @@ Whenever a runtime component such as {CENTRAL} is about to process the KJAR, it
 
 Deployment descriptors supplement the `kmodule.xml` file and provide granular control over your deployment. The presence of these descriptors is optional and your deployment will proceed successfully without them. You can set purely technical properties using these descriptors, including meta values such as persistence, auditing, and runtime strategy.
 
-These descriptors allow you to configure the {KIE_SERVER} on multiple levels (server level default, different deployment descriptor per KJAR, and other server configurations). This allows you to make simple customizations to the default {KIE_SERVER} configuration (possibly per KJAR).
+These descriptors allow you to configure the {KIE_SERVER} on multiple levels (server level default, different deployment descriptor per KJAR, and other server configurations). This enables you to make simple customizations to the default {KIE_SERVER} configuration (possibly per KJAR).
```

Comment:
can we say "... multiple levels, including ... " and list the levels and get rid of the parentheses?

What is "This"? descriptors? or multiple levels?

Reply:
yes, that is much better. "This feature"?

Reply:
These descriptors allow you to configure the {KIE_SERVER} on multiple levels, including server level default, different deployment descriptor per KJAR, and other server configurations. Using descriptors enable you to make simple customizations to the default {KIE_SERVER} configuration, possibly per KJAR.